bin/update

```diff
@@ -18,6 +17,9 @@ chdir APP_ROOT do
   system! 'gem install bundler --conservative'
   system('bundle check') || system!('bundle install')
 
+  # Install JavaScript dependencies if using Yarn
+  # system('bin/yarn')
```

Comment:
Does this comment serve as a friendly reminder of how to install JS dependencies via yarn, should we decide to add it in the future?

Reply:
Yeah - I guess rails is assuming here that you are not using it, but giving you a suggestion of where to run it if you are.